examples/vision/vit_small_ds.py

```diff
@@ -355,21 +354,21 @@ def call(self, encoded_patches):
 """
 
 
-class MultiHeadAttentionLSA(tf.keras.layers.MultiHeadAttention):
+class MultiHeadAttentionLSA(layers.MultiHeadAttention):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         # The trainable temperature term. The initial value is
         # the square root of the key dimension.
         self.tau = tf.Variable(math.sqrt(float(self._key_dim)), trainable=True)
```

Comment:
Please replace this with `keras.Variable()`

Reply:
Done!